kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2697,6 +2698,148 @@ def get_fencer_state(self, req):
+            return jsonobject.dumps(rsp)
+
+        ssh_base = ('ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null '
+                    '-o BatchMode=yes -o ConnectTimeout=5 -p %d root@%s ' % (ssh_port, host_ip))
+
+        def _ssh(remote_cmd, timeout=10):
+            full = "timeout %d %s '%s'" % (timeout, ssh_base, remote_cmd.replace("'", "'\\''"))
+            try:
+                ret, out, err = bash.bash_roe(full)
+                logger.info("[sibling-fence-vm] ssh ret=%s cmd=%s" % (ret, remote_cmd))
+                return ret, out, err
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)